src/makeCalculator.js

```diff
@@ -3,8 +3,42 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
+function makeCalculator(operation, value) {
```

Comment:
[NAMING] - The function `makeCalculator` should not accept parameters `operation` and `value` as they are not used in the function body or in the task description. The function should simply return a calculator object without taking any arguments.